user_management_app/utils.py

```diff
@@ -0,0 +1,25 @@
+from django.http import HttpResponse
+
+
+def get_user_not_found_response():
+    response = HttpResponse("User doesn't exist")
+    response.status_code = 401
+    return response
+
+
+def get_unauthorized_operation_response():
+    response = HttpResponse("Operation not allowed: Unauthorized user")
+    response.status_code = 401
+    return response
+
+
+def is_authorized(request, id):
+    if request.user.id != int(id):
+        return False
+    return True
+
+
+def is_admin(request):
```

Comment:
Since method is to be used by the 'User' model instance so must be defined as property in the model only.